pkg/config/tlsconfig/provider.go

```diff
@@ -35,7 +35,11 @@ func (fn ConfigOptionFunc) apply(c *tls.Config) {
 }
 
 // WithTLSClientAuth sets TLS client authentication options.
-func WithTLSClientAuth(auth tls.ClientAuthType, cas *x509.CertPool, verify func(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error) Option {
+func WithTLSClientAuth(
+	auth tls.ClientAuthType,
```

Comment:
Do we have a clear style here ?

The rules I've been able to infer so far are:
- If you can put all parameters on a line, do it.
- Otherwise put the return and name+receiver on separate lines
- Otherwise do the above, and give each parameter a new line.

Reply:
We don't have a clear style (yet), but here's what I've been doing:

0. If it fits, put it on one line:

```
func (s *Server) Handle(ctx context.Context, proxyReq events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
```

1. If it doesn't fit, split arguments on separate line:

```
func (s *Server) Handle(
	ctx context.Context, proxyReq events.APIGatewayProxyRequest,
) (events.APIGatewayProxyResponse, error) {
```

2. If it still doesn't fit, split arguments onto multiple lines:

```
func (s *Server) Handle(
	ctx context.Context,
	proxyReq events.APIGatewayProxyRequest,
) (events.APIGatewayProxyResponse, error) {
```

3. If it still doesn't fit, and you're sure you shouldn't just rewrite your func, it's also possible to split the return values:

```
func (s *Server) Handle(
	ctx context.Context,
	proxyReq events.APIGatewayProxyRequest,
) (
	events.APIGatewayProxyResponse,
	error,
) {
```